transmogrifier/cli.py

```diff
@@ -49,9 +44,9 @@ def main(source, input_file, output_file, verbose):
     logger.info(configure_sentry())
     logger.info("Running transform for source %s", source)
 
-    input_records = parse_xml_records(input_file)
-    transformer_class = get_transformer(source)
-    transformer_instance = transformer_class(source, input_records)
+    transformer_class = Transformer.get_transformer(source)
+    source_records = transformer_class.parse_source_file(input_file)
+    transformer_instance = transformer_class(source, source_records)
```

Comment:
Perhaps a baby step towards refactoring this could be renaming this variable to just `transformer`, where it's _implied_ this is a `Transformer` instance.

Reply:
Good call!